software/kwallet/en.md

```diff
@@ -31,14 +31,33 @@ The contents of `~/.config/autostart-scripts/ssh-add.sh` should reflect the SSH
 Below is an example of the contents of `.config/autostart-scripts/ssh-add.sh`:
 
 ```
-#!/bin/sh
+#!/bin/bash
 
-ssh-add $HOME/.ssh/id_rsa $HOME/.ssh/solus_id_rsa </dev/null
+ssh-add ssh-add -q < /dev/null
 ```
 
+Tip: The above ssh-add.sh script will only add the default key `~/.ssh/id_rsa`. Assuming you have different keys named key1, key2 etc you need to change the above script to
+
+```
+#!/bin/bash
+
+ssh-add ssh-add ~/.ssh/key1  ~/.ssh/key2 < /dev/null
+```
+
+You also have to set the `SSK_ASKPASS` environment variable to ksshaskpass. For this you have to create the autostart script file and make it executable:
+
+Below is an example of the contents of `.config/plasma-workspace/env/askpass.sh`:
+
+```
+#!/bin/bash
+
+export SSH_ASKPASS='/usr/bin/ksshaskpass'
+```
+
+
 After creating this file, you will need to make it executable:
 
-`chmod a+x ~/.config/autostart-scripts/ssh-add.sh`
+`chmod +x ~/.config/plasma-workspace/env/askpass.sh`
```

Comment:
Why was this changed? a+ ensures the rest of the modes are not affected

Reply:
they guide I read was saying just make it executable that's why I changed it

Reply:
That can be intepreted as "the only action you need to do is make it +x", thus `a+x` would still be accurate.